README.md

```diff
@@ -45,30 +52,79 @@ And ask someone for the values to set the following environment variables:
 - `SUBSCRIPTION_SUCCESS_URL`
 - `WORDPRESS`
 
-#### Run Sheer
-
-  - `sheer index`
-  - `sheer serve`
-
 _You can also add the `export` line to your `.bash_profile`,
 or use your favorite alternative method of setting environment variables._
 
-### Front end
+### Front end setup
+
+The cfgov-refresh front end currently uses the following:
+
+- [Less](http://lesscss.org/)
+- [Capital Framework](https://cfpb.github.io/capital-framework/)
+- [Grunt](http://gruntjs.com/)
+- [Bower](http://bower.io/) & [npm](https://www.npmjs.org/) for package management
+
+If you're new to Capital Framework, we encourage you to
+[start here](https://cfpb.github.io/capital-framework/).
+
+#### Installing dependencies (one time)
+
+1. Install [node.js](http://nodejs.org/) however you'd like.
+2. Install [Grunt](http://gruntjs.com/), [Bower](http://bower.io/),
+   and [Browserify](http://browserify.org/):
```

Comment:
Mmm, probably not. Missed that when editing down the stuff I copied from the OAH README.